app/views/contexts/summary/list_collector_content_block.py

```diff
@@ -0,0 +1,148 @@
+from typing import Any, Iterable, Mapping, MutableMapping, Sequence
+
+from werkzeug.datastructures import ImmutableDict
+
+from app.data_models import AnswerStore, ProgressStore
+from app.data_models.list_store import ListModel, ListStore
+from app.data_models.metadata_proxy import MetadataProxy
+from app.questionnaire import Location, QuestionnaireSchema
+from app.questionnaire.placeholder_renderer import PlaceholderRenderer
+from app.views.contexts.list_content_context import ListContentContext
+from app.views.contexts.summary.block import Block
+
+
+class ListCollectorContentBlock:
+    def __init__(
+        self,
+        routing_path_block_ids: Iterable[str],
+        answer_store: AnswerStore,
+        list_store: ListStore,
+        progress_store: ProgressStore,
```

Comment:
again I think duplication could be lower, either reusing existing block or both inheriting from a base class that already has things like `_get_related_anwers`

Reply:
Ok, made a base class now that both inherit from, thanks!